stuttgart-media-university-author-date.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="de-DE" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Hochschule der Medien Stuttgart (author-date; German)</title>
```

Comment:
> (author-date; German)

@bwiernik, we don't use semicolons as delimiters in titles.

This PR should also update the title of 
https://github.com/citation-style-language/styles/blob/master/stuttgart-media-university.csl to clarify that that is the note variant.